addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -1346,6 +1353,9 @@ export class Rtc extends Record {
                 closeStream(session.videoStreams.get(type));
             }
             session.videoStreams.delete(type);
+            if (this.state.channel.activeRtcSession === this.selfSession) {
+                this.state.channel.activeRtcSession = undefined;
+            }
```

Comment:
This doesn't take into account when camera and screen-sharing are both enabled: disabling one or the other should not remove the `activeRtcSession` when the other one is still present.